### ISSUE TITLE
Function after opening archived web page

### ISSUE THREAD

**Issue by pierrebonbon** [scallywag/org-board#18]
I'm attempting  to open archived web pages with [Hypothesis](https://hypothes.is) automatically enabled, which allows annotating web pages. One way to do it is to inject a script (same as their [bookmarklet](https://web.hypothes.is/start/)). If I get it straight, this should be easy to implement if there was an equivalent of `org-board-run-after-archive-function` for `org-board-open`. Any chance that this could be implemented?

**Comment by scallywag** [comment]
Yes, we can add an abnormal hook (`org-board-after-open-functions`) that runs at the end of `org-board-open`.  Then you should be able to add a function to that hook which will inject the Hypothesis script into the newly opened browser tab.

**Comment by pierrebonbon** [comment]
Thanks for taking the time to answer my question. I would love to help to attempt to implement that hook but, unfortunately, my Elisp skills are close to non-existent so I'm pretty much useless in this regard. Despite that, what can I do to help?
